site/src/components/pages/hub/BlockDataContainer.tsx

```diff
@@ -1,19 +1,14 @@
 import { Box, Tabs, Tab, useTheme, useMediaQuery } from "@mui/material";
-import { BlockProtocolUpdateEntitiesFunction } from "blockprotocol";
 import { Validator } from "jsonschema";
-import { useCallback, useMemo, useState, VoidFunctionComponent } from "react";
+import { useMemo, useState, VoidFunctionComponent } from "react";
+import MockBlockDock from "mock-block-dock";
```

Comment:
```suggestion
import { MockBlockDock } from "mock-block-dock";
```
(if it’s not too late)